data-plane-framework/draft-ietf-detnet-data-plane-framework.xml

```diff
@@ -258,66 +261,99 @@
 	 header or encapsulation technology e.g. <xref target="dn_svcs"/> or it
 	 may be achieved by other standardized technologies e.g.  <xref
 		 target="dn_svc_encaps"/>.
-	
          </t>
          <t>
 	 Extensibility of that Data Plane:  Whether or not the DetNet data plane
 	 includes the facility to carry additional information (metadata) that
 	 can be used to provide an enhanced service to the DetNet packet.
+          </t>
+        </list>
+         </t>
+         <t>
+          (Below replacement text suggested by Balazs)
+         </t>
+         <t>
+         <list style='numbers'>
+         <t>
+          Forwarding paradigm: DetNet can be built over packet switched (IP)
+          networks or label switched (MPLS) networks .  
+         </t>
+         <t>
+          Encapsulation format: DetNet encodes specific flow attributes
+          (namely flow identity and sequence number) in packets. In DetNet IP, 
+          zero encapsulation may be used and no sequence number
+          is available. In MPLS, DetNet specific information may be added
+          explicitly to the packets in the format of S-label and d-CW.  
+         </t>
+         </list>
+         </t>
+
         <figure anchor="dn_svcs" align="center"
         title="DetNet Services">
         <artwork align="center"><![CDATA[
 
 
-DetNet        +-------+  +---------+
-Services      | DN IP |  | DN MPLS |
-              +-------+  +---------+
+  +-------+  +---------+
+  | DN IP |  | DN MPLS |
+  +-------+  +---------+
 
         ]]></artwork>
         </figure>
         <figure anchor="dn_svc_encaps" align="center"
-        title="DetNet Service Encapsulations">
+        title="DetNet Service Examples with Recursion">

```

Comment:
can we drop "with recursion" I think this is confusing.

Reply:
Cheanged